networkx/readwrite/gml.py

```diff
@@ -153,12 +153,12 @@ def read_gml(path, label='label', destringizer=None):
 
     label : string, optional
         If not None, the parsed nodes will be renamed according to node
-        attributes indicated by ``label``. Default value: ``'label'``.
+        attributes indicated by `label`. Default value: `'label'`.
```

Comment:
The link you give for showing quotes doesn't use a special notation for the quoted literal. At least even when I zoom in, I can't tell a difference in font or size.

In numpy's own files ([one example](https://github.com/numpy/numpy/blob/master/numpy/core/numeric.py)), they seem to leave literal strings with no formatting.  So 'F' and 'C' and 'valid' and 'full' express the string literals in the docs. This seems fairly clear and maybe more clear in the raw text than `'full'` or ``'full'``.  So I'd like to go with string literals not getting back-ticks unless they are part of a bigger piece of code.